pkg/virtctl/vm/vm.go

```diff
@@ -193,6 +242,45 @@ func (o *Command) Run(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return fmt.Errorf("Error renaming VirtualMachine %v", err)
 		}
+	case COMMAND_GUESTOSINFO:
+		guestosinfo, err := virtClient.VirtualMachineInstance(namespace).GuestOsInfo(vmiName)
+		if err != nil {
+			return fmt.Errorf("Error renaming VirtualMachine %v", err)
+		}
+
+		data, err := json.MarshalIndent(guestosinfo, "", "  ")
+		if err != nil {
+			return fmt.Errorf("Cannot unmarshal guestosinfo %v", err)
```

Comment:
The above "marshal" the data. Why did you used "unmarshal"?
Also, if the error is reported, it will be unclear what "marshal" is it all about (unless the error gives enough info, then this message is not really useful). If the err does not include the value of `guestosinfo`, it may be useful to add it.

Also note that it is not common to use capital letter in such messages because it is usually appended to other text.
(true for all other messages)